app/views/rooms/index.html.haml

```diff
@@ -0,0 +1,13 @@
+.container
+  %h2 ルーム一覧
+  - @rooms.each do |room|
+    .room-content
+      = link_to room do
+        = room.name
+      = "#{room.reviewees.count} / #{room.capacity}"
+      = room.reviewer.name
+      - if room.reviewer == current_user
+        = link_to "ルームを編集する", edit_room_path(room)
+        = link_to "ルームを削除する", room, method: :delete, data: { confirm: "#{room.name}を削除してもよろしいですか？" }
```

Comment:
__[HAML-Lint]__

Line is too long. [100/80]


<!-- Comment by [SideCI](https://sideci.com) -->